storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerTest.java

```diff
@@ -66,6 +70,17 @@ public TopicBasedRemoteLogMetadataManager topicBasedRlmm() {
         return remoteLogMetadataManagerHarness.remoteLogMetadataManager();
     }
 
+    @Test
+    public void testInternalTopicExists() {
+        Properties adminConfig = remoteLogMetadataManagerHarness.adminClientConfig();
+        ListenerName listenerName = remoteLogMetadataManagerHarness.listenerName();
+        try (Admin admin = remoteLogMetadataManagerHarness.createAdminClient(listenerName, adminConfig)) {
+            NewTopic newTopic = topicBasedRlmm().createRemoteLogMetadataTopicRequest();
```

Comment:
Can not we use `topicBasedRlmm().config().remoteLogMetadataTopicName()` instead of createRemoteLogMetadataTopicRequest() method?